### PR TITLE
Monteserrat font and color variable added

### PR DESCRIPTION
**Pull request by fricto**

### CODE REVIEW

File: ui.frontend/src/main/webpack/site/_variables.scss [hunk 1]
```diff
@@ -2,6 +2,7 @@
 //== Font
 
 $font-family:           'Montserrat', sans-serif;
+$font-size:             16px;
```
Comment (fricto) [inline ui.frontend/src/main/webpack/site/_variables.scss]: Should we keep with the 'rem' version?

File: ui.frontend/src/main/webpack/site/main.scss [hunk 2]
```diff
@@ -1,6 +1,7 @@
-@import url('https://fonts.googleapis.com/css?family=Montserrat:300,400,600,700,800');
+@import url('https://fonts.googleapis.com/css?family=Montserrat:300,400,500,600,700,800,900');
```
Comment (fricto) [inline ui.frontend/src/main/webpack/site/main.scss]: Do we need all these weights?

**Review by fricto — COMMENTED** [review]
Please see questions

File: ui.frontend/src/main/webpack/site/main.scss (hunk 2 above)
Comment (SushmaGarg-ICFNext) [reply ui.frontend/src/main/webpack/site/main.scss]: i am guessing auto merge added all again(i removed 500-medium and 900-black). We pretty much need 4 variants-- 300(lighter), 400(regular), 600(semibold) and 700(bold) as per screens provided.

File: ui.frontend/src/main/webpack/site/_variables.scss (hunk 1 above)
Comment (SushmaGarg-ICFNext) [reply ui.frontend/src/main/webpack/site/_variables.scss]: i am guessing auto merge added both. i converted px to rem, so just rem should be fine. Will make that change.

File: ui.frontend/src/main/webpack/site/main.scss (hunk 2 above)
Comment (SushmaGarg-ICFNext) [reply ui.frontend/src/main/webpack/site/main.scss]: changes added in next commit

File: ui.frontend/src/main/webpack/site/_variables.scss (hunk 1 above)
Comment (SushmaGarg-ICFNext) [reply ui.frontend/src/main/webpack/site/_variables.scss]: px version removed in next commit

**Pull request merged**